src/DataViews/include/DataViews/DataView.h

```diff
@@ -58,12 +61,25 @@ constexpr std::string_view kMenuActionCopySelection = "Copy Selection";
 constexpr std::string_view kMenuActionExportToCsv = "Export to CSV";
 constexpr std::string_view kMenuActionExportEventsToCsv = "Export events to CSV";
 
+static constexpr const char* kFieldSeparator = ",";
+// CSV RFC requires lines to end with CRLF
+static constexpr const char* kLineSeparator = "\r\n";
```

Comment:
Could you define these as `constexpr std::string_view` - similar to the other constants?